views/forms/ControlledMultiSelect/index.jsx

```diff
@@ -11,10 +27,22 @@ const ControlledMultiSelect = (props) => {
         <Autocomplete
           id={props.name}
           fullWidth={props.fullWidth}
+          getOptionDisabled={(option) => option.isFixed}
           isOptionEqualToValue={(option, value) => option.value === value.value}
           multiple
-          onChange={(_, data) => field.onChange(data)}
+          onChange={(e, data, reason) => {
+            onChange(e, data, reason, field)
+          }}
           options={props.options}
+          renderTags={(tagValue, getTagProps) =>
+            tagValue.map((option, index) => (
+              <Chip
+                label={option.label}
+                {...getTagProps({ index })}
+                disabled={option.isFixed}
+              />
+            ))
+          }
```

Comment:
do you need this if you also have the `getOptionDisabled` prop?

Reply:
yes getOptionDisabled is to disable selection on dropdown, the disabled option on the Chip component is to disable removing or deleting the selection from the input bar